src/renderer/views/Watch/Watch.js

```diff
@@ -299,7 +304,17 @@ export default Vue.extend({
           })
 
           this.videoPublished = new Date(result.videoDetails.publishDate.replace('-', '/')).getTime()
-          this.videoDescription = result.player_response.videoDetails.shortDescription
+          try {
+            // workaround for description localization
+            const descriptionLines = result.response.contents.twoColumnWatchNextResults.results.results.contents[1].videoSecondaryInfoRenderer.description.runs
+            this.videoDescription = ''
+            for (let i = 0; i < descriptionLines.length; i++) {
+              this.videoDescription += `${descriptionLines[i].text}\n`
+            }
```

Comment:
this can be simplified by using [Array#join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)
```suggestion
            this.videoDescription = descriptionLines?.join('\n') ?? ''
```